tests/components/komfovent/test_config_flow.py

```diff
@@ -0,0 +1,102 @@
+"""Test the Komfovent config flow."""
```

Comment:
I'm missing the test that checks if the flow aborts if the unique id is already configured